src/Screens/Assignment3/Components/FriendStatus/FriendStatus.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { AuthButton } from "../../../../Components/Atoms/AuthButton/AuthButton";
+import { SignIn, SignOut } from "../../../../Constants/Constants";
+import { useCheckFriendStatus } from "../../../../Hooks/useCheckFriendStatus";
+import { Div, Heading } from "./FriendStatusStyle";
+
+export const FriendStatus = () => {
+  const [isOnline, setIsOnline] = useState<boolean>(false);
+  const friendStatus = useCheckFriendStatus(isOnline);
+
+  const handleStatusChange = () => {
+    setIsOnline(!isOnline);
+  };
+
+  return (
+    <Div>
```

Comment:
Please use different names of style components i.e. not related to existing html tags